common/web/keyman-version/build.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Builds the include script for the current Keyman version.
+#
+
+# Exit on command failure and when using unset variables:
+set -eu
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../../../resources/build/build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+# This script runs from its own folder
+cd "$(dirname "$THIS_SCRIPT")"
+
+display_usage ( ) {
+  echo "Usage: $0 [configure] [clean] [build] [test]"
+  echo "          [--verbose|-v]"
+  echo "       $0 -h|--help"
+  echo
+  echo "  clean                  removes build/ folder"
+  echo "  configure              runs 'npm ci' on root folder"
+  echo "  build                  builds wrapped version of package"
+  echo "                           [if required will: configure]"
+  echo "  test                   runs tests (builds as req'd)"
+  echo "                           [if required will: build]"
+}
+
+################################ Main script ################################
+
+builder_init "clean configure build" "$@"
+
+# TODO: build if out-of-date if test is specified
+# TODO: configure if npm has not been run, and build is specified
+
+if builder_has_action configure; then
+  verify_npm_setup
+  builder_report configure success
+fi
+
+if builder_has_action clean; then
+  npm run clean
+  rm -f ./version.inc.ts
+  builder_report clean success
+fi
+
+if builder_has_action build; then
+  # Generate index.ts
+  echo "
+    // Generated by common/web/keyman-version/build.sh
+    namespace KEYMAN_VERSION {
```

Comment:
Ooh, a new top-level namespace for KeymanWeb?  Well, I guess probably not _quite_ yet, since this is only used for module-compatible projects (the lexical model compiler, which is Developer only) at the moment in this PR chain.  But the second it's linked into Web, it'll be on the same level as the whole `com` & `com.keyman` thing you've voiced displeasure about before.

Keep in mind that Web proper can't do node-style require statements.  Anything module-based will avoid the new namespace easily because of how `module.exports` works, fortunately.